pkg/reconciler/buildrun/buildrun.go

```diff
@@ -126,6 +126,7 @@ func (r *ReconcileBuildRun) Reconcile(request reconcile.Request) (reconcile.Resu
 			// Set OwnerReference for Build and BuildRun only when build.shipwright.io/build-run-deletion is set "true"
 			if build.GetAnnotations()[buildv1alpha1.AnnotationBuildRunDeletion] == "true" && !resources.IsOwnedByBuild(build, buildRun.OwnerReferences) {
 				if err := r.setOwnerReferenceFunc(build, buildRun, r.scheme); err != nil {
+					// TODO - why are we setting this on the Build object instead of the BuildRun object?
```

Comment:
I personally prefer that issues are opened instead of adding TODOs in unrelated pull requests that nobody will work on.

Reply:
Opened #827 - I'll drop this comment